RxSwift/Observables/Observable+StandardSequenceOperators.swift

```diff
@@ -77,7 +77,30 @@ extension ObservableType {
         }
     }
 }
+
+// MARK: takeLast
+
+extension ObservableType {
     
+    /**
+    Returns a specified number of the last elements before the observable sequence completed.
+     
+     - parameter count: The number of elements to return.
+     - returns: An observable sequence that contains the specified number of elements from the end of the input sequence.
+     */
```

Comment:
Could you please copy the documentation from

https://github.com/Reactive-Extensions/RxJS/blob/8f12f812d497acf639588e90f74d504a9fc801ec/src/core/linq/observable/takelast.js#L27

... to be consistent :)